data/hai/unfettered side missions.txt

```diff
@@ -160,267 +96,413 @@ mission "Hai Legacy 1"
 				die
 			label lucky?
-			`	Probably thanks to the fact the ship's shields are down, and of its depreciated state, you manage to land a very lucky shot, going trough a layer of rust and severely damaging the reactor of the ship.`
-			`	At least, it might have seemed lucky until it blew up, taking you and everyone else on this beach down with it.`
+			`	Probably thanks to the fact the ship's shields are down, and its depreciated state, you manage to land a very lucky shot, going through a layer of rust and severely damaging the reactor of the ship. The ship's speakers cry out, "MY REACTOR... NOT MY REACTOR!"`
+			`	It might have seemed lucky, that is, until it blows up, taking you and everyone else on the beach down with it.`
 				die
+			label miss
+			`	You miss your shot. Neither the ship nor the two Hai react, probably not viewing you as a threat. You still have time to do something else or try again. You feel like you heard a chuckle, but you're not sure where it came from.`
+				goto "try again"
 	on defer
 		set "Hai Legacy 1: deferred"
+	on accept
+		clear "unfettered: ocean dialog"
+		clear "unfettered: pilot dialog"
 
 mission "Hai Legacy 2"
 	autosave
 	landing
 	name "Heritage reclaimed"
-	description "Use the newly acquired Ancient Ladybug to bring aid to the Unfettered Hai, with it as your flagship and preferably without any other ships."
+	description "Use the newly acquired Anomalocaris to bring aid to the Unfettered Hai on <planet> by <date>, with it as your flagship and preferably without any other ships."
 	clearance
 	# deadline to make sure you don't just enjoy free rep
 	deadline 30
 	cargo "food (aid)" 121
 	source "Hai-home"
 	destination "Firelode"
-	blocked "You need space for the agreed rush cargo of food to Unfettered space on the Ancient Ladybug, and should use it as flagship, preferably as your only ship."
 	to offer
 		not "Hai Legacy 1: declined"
+		has "First Contact: Unfettered: offered"
+		"reputation: Hai (Unfettered)" < 0
+		not "Unfettered Jump Drive 1: offered"
+		"credits" >= 10000000
 	to accept
-		has "flagship model: Ancient Ladybug"
+		has "ship model: Anomalocaris"
 	on offer
 		conversation
-			`Going into the spaceport one could say the cargo finds you more than you found it: a worker comes to you with crates, which are already allocated in your name for this ship. All that is left to do is to bring it to <destination> for the big promised sum of <payment>.`
+			`Upon returning to the spaceport, one could say the cargo finds you more than you find it. A worker immediately greets you while others begin to fill your ship with crates, which a quick check shows have already been allocated for this ship in your name. All that is left to do is to bring it to <destination> by <day> for the promised sum of <payment>.`
```

Comment:
Is this `<day>` instead of `<date>` on purpose?

Reply:
```suggestion
			`Upon returning to the spaceport, one could say the cargo finds you more than you find it. A worker immediately greets you while others begin to fill your ship with crates, which a quick check shows have already been allocated for this ship in your name. All that is left to do is to bring it to <destination> by <date> for the promised sum of <payment>.`
```
not really